Dockerfile

```diff
@@ -1,36 +1,15 @@
 ARG NGINX_IMAGE_TAG=stable-alpine-for-hugo
 
-FROM debian:10-slim AS builder
-
-ARG HUGO_VERSION=0.76.5
-ARG NODE_VERSION=12.18.3
-
-RUN apt-get update && apt-get install -y \
-    build-essential \
-    ca-certificates \
-    curl \
-    git \
-    --no-install-recommends
-
-RUN curl -L -o /tmp/node.tar.xz "https://nodejs.org/dist/v${NODE_VERSION}/node-v${NODE_VERSION}-linux-x64.tar.xz" \
-    && mkdir -p /usr/local/lib/nodejs \
-    && tar -xJf /tmp/node.tar.xz -C /usr/local/lib/nodejs \
-    && ln -s /usr/local/lib/nodejs/node-v${NODE_VERSION}-linux-x64/bin/node /usr/bin/node \
-    && ln -s /usr/local/lib/nodejs/node-v${NODE_VERSION}-linux-x64/bin/npm /usr/bin/npm \
-    && ln -s /usr/local/lib/nodejs/node-v${NODE_VERSION}-linux-x64/bin/npx /usr/bin/npx
-
-RUN curl -L -o /tmp/hugo.deb "https://github.com/gohugoio/hugo/releases/download/v${HUGO_VERSION}/hugo_${HUGO_VERSION}_Linux-64bit.deb" \
-    && dpkg -i /tmp/hugo.deb
+FROM eclipsefdn/hugo-node:latest AS builder
+# See https://github.com/EclipseFdn/dockerfiles/tree/master/hugo-node 
+# and https://github.com/EclipseFdn/dockerfiles/blob/master/build.sh
+# for hugo and node versions (note the latest in the build.sh file)
 
 WORKDIR /workdir
 COPY . /workdir/
 
-RUN echo -n "Node.js version " && node --version \
-  && echo -n "npm version " && npm --version \
-  && hugo version
-
 RUN npm --unsafe-perm ci \
-    && hugo --gc --minify
+    && hugo
```

Comment:
I would have prefered that we continue to minify the output of our HTML files..

Reply:
This was the only website using `--gc --minify` so I though it was better to align with others. Note that I removed a little bit more of clutter. The `Dockerfile` is not defined anymore in each website's repository, but in [a central location](https://github.com/EclipseFdn/jenkins-pipeline-shared/blob/master/resources/org/eclipsefdn/hugoWebsite/Dockerfile). One change is enough to modify all websites. See how the new centralized `Dockerfile` now has `--gc --minify`